sensor_msgs/Image image
---
float64 pepper_blob_center_of_gravity_x
float64 pepper_blob_center_of_gravity_y
float64 pepper_blob_size